optional/e2e/features/home.feature

```diff
@@ -2,22 +2,30 @@
 
 @home
 Fonctionnalité: Page d'acceuil
-  Pour pouvoir trouver une information sur le droit du travai
+  Pour pouvoir trouver une information sur le droit du travail
   En tant que visiteur
-  Je veux être pouvoir consulter la page d'acceuil
+  Je veux pouvoir consulter la page d'acceuil
 
 Scénario:
-  Soit un navigateur web sur le site
+  Soit un utilisateur sur la page d'accueil
   Alors je vois "Bienvenue sur"
   Alors je vois "le Code du travail numérique"
-  Alors je vois "Obtenez des réponses aux questions que vous vous posez sur le droit du travail."
-  Alors je vois "Rechercher"
+  Alors je vois "Recherchez par mots-clés"
+  Alors je vois le bouton "Rechercher"
 
-  Alors je vois "Boîte à outils"
-  Alors je vois "Convention collective"
-  Alors je vois "Préavis de démission"
-  Alors je vois "Salaire brut/net"
-  Alors je vois "Modèles de documents"
+  Alors je vois "4" tuiles sous le "h2" "Boîte à outils"
+  Alors le lien "Voir tous les outils" pointe sur "outils"
 
-  Alors je vois "Besoin de plus d’information"
-  Alors je vois "Contacter nos services en région"
+  Alors je vois "Thèmes"
+  Alors je vois "Embauche et contrat de travail"
```

Comment:
I wonder if we should assert precisely on content like this since it implies to update all the test when we change content

Reply:
We were wondering is counting the tiles was enough in this case
@virginielastisneres what do you think ?